www/pages/index.tsx

```diff
@@ -39,53 +39,55 @@ function IndexPage(props: PageProps) {
     .filter((rule: RuleData) => rule.code.includes(search));
 
   return (
-    <div class={tw`mx-auto max-w-screen-md px-6 sm:px-6 md:px-8`}>
-      <Head>
-        <link
-          rel="stylesheet"
-          href="https://cdn.jsdelivr.net/gh/lucacasonato/manual@df7ae27/www/static/markdown.css"
-          crossOrigin="anonymous"
-        />
-        <style>{style}</style>
-      </Head>
-      <Header />
-      <main class={tw`my-8`}>
-        <label for="search" class={tw`sr-only`}>Search</label>
-        <form id="search_form">
-          <input
-            type="text"
-            name="q"
-            class={tw`w-full border h-10 border-gray-200 rounded rounded-r-none px-3 relative`}
-            placeholder="Search"
-            value={search}
+    <div class={tw`dark:bg-[#0d1117] dark:text-white py-6`}>
+      <div class={tw`mx-auto max-w-screen-md px-6 sm:px-6 md:px-8`}>
+        <Head>
+          <link
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/gh/lucacasonato/manual@df7ae27/www/static/markdown.css"
+            crossOrigin="anonymous"
           />
-          <div class={tw`mt-2`}>
+          <style>{style}</style>
+        </Head>
+        <Header />
+        <main class={tw`my-8`}>
+          <label for="search" class={tw`sr-only`}>Search</label>
+          <form id="search_form">
             <input
-              type="checkbox"
-              id="all_rules"
-              name="all"
-              checked={allRules}
+              type="text"
+              name="q"
+              class={tw`w-full border h-10 border-gray-200 dark:border-gray-500 rounded rounded-r-none px-3 relative dark:bg-gray-800 `}
+              placeholder="Search"
+              value={search}
             />
-            <label htmlFor="all_rules" class={tw`ml-2`}>
-              Show all rules
-            </label>
+            <div class={tw`mt-2`}>
+              <input
+                type="checkbox"
+                id="all_rules"
+                name="all"
+                checked={allRules}
+              />
+              <label htmlFor="all_rules" class={tw`ml-2`}>
+                Show all rules
+              </label>
+            </div>
+          </form>
+          <script
+            dangerouslySetInnerHTML={{
+              __html:
+                "document.getElementById('all_rules').oninput = () => document.getElementById('search_form').submit();",
+            }}
+          >
+          </script>
+          <div class={tw`mt-6 text-gray-600`}>
```

Comment:
```suggestion
          <div class={tw`mt-6 text-gray-600 dark:text-gray-400`}>
```

Not enough contrast here:
![image](https://user-images.githubusercontent.com/3132889/185976014-87667e1a-7e1e-4353-908e-a116a8757f09.png)